### DISCUSSION TITLE
The storage servers wen't down

### DISCUSSION THREAD

**Discussion by TheBlueOompaLoompa** [supabase/supabase#1483] [Questions] [answered]
Everything in my storage bucket won't load right now.
![image](https://user-images.githubusercontent.com/53215039/117524925-7b5aee00-af85-11eb-947e-129abaf44953.png)

**Comment by inian** [comment]
Can you reach out to beta@supabase.io with your project reference @TheBlueOompaLoompa ?

**Reply by inian** [reply]
Ah I see. You have already reached out! The issue should have been sorted by now, lets continue the conversation there.

**Comment by TheBlueOompaLoompa** [comment]
<https://app.supabase.io/project/ubzhjfzgitrcbnsedlpp>

**Comment by TheBlueOompaLoompa** [comment, marked as answer]
Everything is working now, thanks! :)
